packages/runtime/src/enhancements/node/json-processor.ts

```
@@ -0,0 +1,92 @@
+            for (const [key, value] of Object.entries(item)) {
+                const fieldInfo = resolveField(this.options.modelMeta, typeDef, key, true);
+                if (!fieldInfo) {
+                    continue;
+                }
+                if (fieldInfo.isTypeDef) {
+                    // recurse
+                    this.fixJsonDateFields(value, fieldInfo.type);
+                } else if (fieldInfo.type === 'DateTime' && typeof value === 'string') {
+                    // convert to Date
+                    const parsed = Date.parse(value);
+                    if (!isNaN(parsed)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)